opencolorio_config_aces/__init__.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+OpenColorIO Configuration for ACES
+==================================
+
+The `OpenColorIO Configuration for ACES \
+<https://github.com/AcademySoftwareFoundation/OpenColorIO-Config-ACES/>`__ is
+an open-source `Python <https://www.python.org/>`__ package implementing
+support for the generation of the *OCIO* configuration for the
+*Academy Color Encoding System* (ACES).
+
+It is freely available under the
+`New BSD License <https://opensource.org/licenses/BSD-3-Clause>`__ terms.
+
+Sub-packages
+------------
+-   config: Objects implementing support for the *OCIO* config generation.
+-   utilities: Various utilities and data structures.
+"""
+
+from .config import (
+    discover_aces_ctl_transforms, classify_aces_ctl_transforms,
+    unclassify_ctl_transforms, filter_ctl_transforms, print_aces_taxonomy,
+    build_aces_conversion_graph, node_to_ctl_transform, ctl_transform_to_node,
+    filter_nodes, plot_aces_conversion_graph)
+
+__author__ = 'OpenColorIO-Config-ACES Contributors'
```

Comment:
More to discuss, but we should consider using the recommended LF license header formatting like OCIO. This would involve every file having one of two license statements:

For source files like this one:
```
# SPDX-License-Identifier: BSD-3-Clause
# Copyright Contributors to the OpenColorIO Project.
```

And for documentation and text files (comment syntax varying by language, *.rst example below):
```
..
  SPDX-License-Identifier: CC-BY-4.0
  Copyright Contributors to the OpenColorIO Project.
```

Reply:
And this could still be supplemented by the standard dunder variables of course.

Reply:
Yep will do a pass to fix all that once we settle on the licensing!